packages/react-composites/tests/browser/common/utils.ts

```diff
@@ -393,18 +394,43 @@ export const buildUrl = (
 ): string => `${serverUrl}?${encodeQueryData({ ...user, ...qArgs })}`;
 
 // Unexported types from @playwright/tests package we need
-type PageFunction<R> = string | ((arg: unknown) => R | Promise<R>);
-type SmartHandle<T> = T extends Node ? ElementHandle<T> : JSHandle<T>;
+export type NoHandles<Arg> = Arg extends JSHandle
+  ? never
+  : Arg extends object
+    ? { [Key in keyof Arg]: NoHandles<Arg[Key]> }
+    : Arg;
+export type Unboxed<Arg> =
```

Comment:
Can we do anything to simplify `Unboxed<Arg>` type?

Reply:
This is copied from Playwright.